WARNING:root:mensaje a nivel warning
INFO:root:mensaje a nivel info
DEBUG:root:mensaje a nivel debug
ERROR:root:Ocurrio un error en la base de datos
2021-05-20 09:22:56,562: WARNING 
2021-05-20 09:22:56,563: INFO 
2021-05-20 09:22:56,564: DEBUG 
2021-05-20 09:22:56,565: ERROR 
2021-05-20 09:24:01,537: WARNING 
2021-05-20 09:24:01,537: INFO 
2021-05-20 09:24:01,538: DEBUG 
2021-05-20 09:24:01,538: ERROR 
2021-05-20 09:30:27,602: WARNING [logger_base.py: 12]
2021-05-20 09:30:27,603: INFO [logger_base.py: 13]
2021-05-20 09:30:27,603: DEBUG [logger_base.py: 14]
2021-05-20 09:30:27,604: ERROR [logger_base.py: 15]
2021-05-20 09:32:19,694: WARNING [logger_base.py: 12] mensaje a nivel warning
2021-05-20 09:32:19,695: INFO [logger_base.py: 13] mensaje a nivel info
2021-05-20 09:32:19,696: DEBUG [logger_base.py: 14] mensaje a nivel debug
2021-05-20 09:32:19,697: ERROR [logger_base.py: 15] Ocurrio un error en la base de datos
09:38:17 AM: WARNING [logger_base.py: 13] mensaje a nivel warning
09:38:17 AM: INFO [logger_base.py: 14] mensaje a nivel info
09:38:17 AM: DEBUG [logger_base.py: 15] mensaje a nivel debug
09:38:17 AM: ERROR [logger_base.py: 16] Ocurrio un error en la base de datos
09:45:10 AM: WARNING [logger_base.py: 14] mensaje a nivel warning
09:45:10 AM: INFO [logger_base.py: 15] mensaje a nivel info
09:45:10 AM: DEBUG [logger_base.py: 16] mensaje a nivel debug
09:45:10 AM: ERROR [logger_base.py: 17] Ocurrio un error en la base de datos
11:05:04 AM: ERROR [conexion.py: 27] Error a conectar a la DB:function takes at most 4 keyword arguments (5 given)
11:08:12 AM: ERROR [conexion.py: 27] Error al conectar a la BD:function takes at most 4 keyword arguments (5 given)
11:13:32 AM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x00D0F5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:13:32 AM: INFO [conexion.py: 34] <connection object at 0x00D0F5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:33:11 AM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x00D9F5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:33:11 AM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x0353D360; closed: 0>
11:33:11 AM: INFO [conexion.py: 44] <cursor object at 0x0353D360; closed: 0>
11:47:12 AM: DEBUG [conexion.py: 57] Se han cerrado los objetos de cursor y conexion
11:47:13 AM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x009DF5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:47:13 AM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x0319D360; closed: 0>
11:47:13 AM: INFO [conexion.py: 61] <cursor object at 0x0319D360; closed: 0>
11:51:00 AM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
11:51:00 AM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x0301F5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:51:00 AM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x035ED2D8; closed: 0>
11:51:00 AM: INFO [conexion.py: 63] <cursor object at 0x035ED2D8; closed: 0>
01:42:55 PM: DEBUG [persona.py: 22] Id Persona: 1, Nombre: Juan, Apellido: Ruffo, Email: Jruffo@mail.com 
01:45:29 PM: DEBUG [persona.py: 22] Id Persona: 1, Nombre: Juan, Apellido: Ruffo, Email: Jruffo@mail.com 
01:45:29 PM: DEBUG [persona.py: 25] Id Persona: None, Nombre: Karla, Apellido: Gomez, Email: Kgomez@mail.com 
01:51:51 PM: DEBUG [persona.py: 22] Id Persona: 1, Nombre: Juan, Apellido: Ruffo, Email: Jruffo@mail.com 
01:51:51 PM: DEBUG [persona.py: 25] Id Persona: None, Nombre: Karla, Apellido: Gomez, Email: Kgomez@mail.com 
01:51:51 PM: DEBUG [persona.py: 28] Id Persona: 3, Nombre: None, Apellido: None, Email: None 
03:14:09 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:17:18 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:18:29 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:18:29 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x02E89490; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:19:54 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:19:54 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x02E49490; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:21:11 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:21:11 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x03C8D8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:32:23 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:32:24 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x0327E8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:34:34 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:34:34 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x030DD8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:36:19 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:36:19 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x034AD8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:40:22 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:40:22 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x034ED8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:40:22 PM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x0370E3E8; closed: 0>
03:41:01 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:41:01 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x01D7D8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:41:01 PM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x03CFE3E8; closed: 0>
03:41:59 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:41:59 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x0392D8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:41:59 PM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x03B3E3E8; closed: 0>
03:45:13 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:45:13 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x0367D8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:45:13 PM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x0386E3E8; closed: 0>
03:45:13 PM: DEBUG [persona_dao.py: 16] b'SELECT * FROM persona ORDER BY id_persona'
03:45:35 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:46:25 PM: DEBUG [conexion.py: 59] Se han cerrado los objetos de cursor y conexion
03:46:25 PM: DEBUG [conexion.py: 24] Conexion exitosa:<connection object at 0x014DF5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:46:25 PM: DEBUG [conexion.py: 37] Se abrio el cursor con exito:<cursor object at 0x035CD2D8; closed: 0>
03:46:25 PM: INFO [conexion.py: 63] <cursor object at 0x035CD2D8; closed: 0>
03:48:31 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x0102F5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:48:31 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x0380D2D8; closed: 0>
03:48:31 PM: INFO [conexion.py: 59] <cursor object at 0x0380D2D8; closed: 0>
03:48:31 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
03:48:52 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x018ED8F0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
03:48:52 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x0386D360; closed: 0>
03:48:52 PM: DEBUG [persona_dao.py: 16] b'SELECT * FROM persona ORDER BY id_persona'
03:48:52 PM: DEBUG [persona_dao.py: 30] Id Persona: 1, Nombre: Juan, Apellido: Perez, Email: Jperez@mail.com 
03:48:52 PM: DEBUG [persona_dao.py: 30] Id Persona: 2, Nombre: Karla1, Apellido: Gomez2, Email: Kgomez21@mail.com 
03:48:52 PM: DEBUG [persona_dao.py: 30] Id Persona: 4, Nombre: Carlos, Apellido: Reinoza, Email: Creinoza@mail.com 
03:48:52 PM: DEBUG [persona_dao.py: 30] Id Persona: 10, Nombre: Mathias, Apellido: Saavedra, Email: Msaavedra@mail.com 
03:48:52 PM: DEBUG [persona_dao.py: 30] Id Persona: 11, Nombre: Maria, Apellido: Longa, Email: Mlonga@mail.com 
03:48:52 PM: DEBUG [persona_dao.py: 30] Id Persona: 12, Nombre: Fernando, Apellido: Ibarra, Email: Fibarra@mail.com 
03:48:52 PM: DEBUG [persona_dao.py: 30] Id Persona: 13, Nombre: Marcos, Apellido: Tunez, Email: Mtunez@mail.com 
04:37:49 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x01B6F5D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
04:37:49 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x03C6E3E8; closed: 0>
04:37:49 PM: DEBUG [persona_dao.py: 33] b'INSERT INTO persona(nombre, apellido, email) VALUES(%s,%S,%S)'
04:37:49 PM: DEBUG [persona_dao.py: 34] Persona a insertar: Id Persona: None, Nombre: Pedro, Apellido: Najera, Email: Pnajera@mail.com 
04:37:49 PM: ERROR [persona_dao.py: 41] Excepción al insertar persona:unsupported format character 'S' (0x53) at index 56
04:37:49 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
04:37:49 PM: DEBUG [persona_dao.py: 54] Personas insertadas: None
01:27:29 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x000001EA27FB07B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:27:29 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x000001EA28385C80; closed: 0>
01:27:29 PM: DEBUG [persona_dao.py: 33] b'INSERT INTO persona(nombre, apellido, email) VALUES(%s,%S,%S)'
01:27:29 PM: DEBUG [persona_dao.py: 34] Persona a insertar: Id Persona: None, Nombre: Pedro, Apellido: Najera, Email: Pnajera@mail.com 
01:27:29 PM: ERROR [persona_dao.py: 41] Excepción al insertar persona:unsupported format character 'S' (0x53) at index %Id
01:27:29 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
01:27:29 PM: DEBUG [persona_dao.py: 54] Personas insertadas: None
01:45:10 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x000001DE67B077B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:45:10 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x000001DE67FA0AC0; closed: 0>
01:45:10 PM: DEBUG [persona_dao.py: 52] b'UPDATE persona SET nombre=%s, apellido=%s, email=%s WHERE id_persona=%s'
01:45:10 PM: DEBUG [persona_dao.py: 53] Persona a actualizar: Id Persona: 1, Nombre: Juan2, Apellido: Perez2, Email: jperez2@mail.com 
01:45:10 PM: ERROR [persona_dao.py: 60] Excepción al actualizar persona:'Persona' object has no attribute 'get_id_persona'
01:45:10 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
01:45:10 PM: DEBUG [persona_dao.py: 95] Personas actualizadas: None
01:45:57 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x000001EC036517B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:45:57 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x000001EC039F5C80; closed: 0>
01:45:57 PM: DEBUG [persona_dao.py: 52] b'UPDATE persona SET nombre=%s, apellido=%s, email=%s WHERE id_persona=%s'
01:45:57 PM: DEBUG [persona_dao.py: 53] Persona a actualizar: Id Persona: 1, Nombre: Juan2, Apellido: Perez2, Email: jperez2@mail.com 
01:45:57 PM: ERROR [persona_dao.py: 60] Excepción al actualizar persona:'Persona' object has no attribute 'get_id_persona'
01:45:57 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
01:45:57 PM: DEBUG [persona_dao.py: 95] Personas actualizadas: None
01:46:26 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x0000016FAF8F17B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:46:26 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x0000016FAFCD5C80; closed: 0>
01:46:26 PM: DEBUG [persona_dao.py: 52] b'UPDATE persona SET nombre=%s, apellido=%s, email=%s WHERE id_persona=%s'
01:46:26 PM: DEBUG [persona_dao.py: 53] Persona a actualizar: Id Persona: 1, Nombre: Juan2, Apellido: Perez2, Email: jperez2@mail.com 
01:46:26 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
01:46:26 PM: DEBUG [persona_dao.py: 95] Personas actualizadas: 1
01:48:12 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x000001983EB277B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:48:12 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x000001983EEE0AC0; closed: 0>
01:48:12 PM: DEBUG [persona_dao.py: 52] b'UPDATE persona SET nombre=%s, apellido=%s, email=%s WHERE id_persona=%s'
01:48:12 PM: DEBUG [persona_dao.py: 53] Persona a actualizar: Id Persona: 1, Nombre: Juan, Apellido: Perez, Email: jperez@mail.com 
01:48:12 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
01:48:12 PM: DEBUG [persona_dao.py: 95] Personas actualizadas: 1
01:49:19 PM: DEBUG [conexion.py: 23] Conexión exitosa: <connection object at 0x00000157B4DF77B0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
01:49:19 PM: DEBUG [conexion.py: 36] Se abrio el cursor con éxito: <cursor object at 0x00000157B51BFAC0; closed: 0>
01:49:19 PM: DEBUG [persona_dao.py: 69] b'DELETE FROM persona WHERE id_persona = %s'
01:49:19 PM: DEBUG [persona_dao.py: 70] Persona a eliminar: Id Persona: 10, Nombre: None, Apellido: None, Email: None 
01:49:19 PM: DEBUG [conexion.py: 56] Se han cerrado los objetos de conexión y cursor
01:49:19 PM: DEBUG [persona_dao.py: 100] Personas eliminadas: 1
09:28:36 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x00000237FE3B3250>
09:28:36 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x00000237FE377590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:30:59 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000018D566D3250>
09:30:59 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x0000018D56697590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:30:59 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x0000018D56ADE150; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:35:50 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001C294675310>
09:35:50 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x000001C2929B7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:35:50 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x000001C294A4E150; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:35:50 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x000001C2929B7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:35:50 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x000001C294675310>
09:35:50 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x000001C294A4E150; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 1>
09:35:50 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x000001C294675310>
09:46:05 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000022F161A5310>
09:46:05 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x0000022F16137590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:46:05 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x0000022F1655E150; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:46:05 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x0000022F16137590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
09:46:05 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x0000022F161A5310>
09:46:05 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x0000022F1655E150; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 1>
09:46:05 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x0000022F161A5310>
09:46:05 AM: DEBUG [conexion.py: 54] Cerramos todas las conexiones del pool: <psycopg2.pool.SimpleConnectionPool object at 0x0000022F161A5310>
10:42:08 AM: DEBUG [cursor_del_pool.py: 11] Inicio de with metodo __enter__ 
10:42:08 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001E8A89E8850>
10:42:08 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x000001E8A8A07590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
10:42:08 AM: DEBUG [cursor_del_pool.py: 18] Se ejecuta el metodo __exit__()
10:42:08 AM: DEBUG [cursor_del_pool.py: 25] Commit de la transaccion
10:42:08 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x000001E8A8A07590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
10:42:08 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x000001E8A89E8850>
10:47:43 AM: DEBUG [cursor_del_pool.py: 11] Inicio de with metodo __enter__ 
10:47:43 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001B7680E8850>
10:47:43 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x000001B769D67590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
10:47:43 AM: DEBUG [cursor_del_pool.py: 37] Listado de personas
10:47:43 AM: DEBUG [cursor_del_pool.py: 38] [(4, 'Carlos', 'Reinoza', 'Creinoza@mail.com'), (2, 'Karla1', 'Gomez2', 'Kgomez21@mail.com'), (11, 'Maria', 'Longa', 'Mlonga@mail.com'), (12, 'Fernando', 'Ibarra', 'Fibarra@mail.com'), (13, 'Marcos', 'Tunez', 'Mtunez@mail.com'), (1, 'Juan', 'Perez', 'jperez@mail.com')]
10:47:43 AM: DEBUG [cursor_del_pool.py: 18] Se ejecuta el metodo __exit__()
10:47:43 AM: DEBUG [cursor_del_pool.py: 25] Commit de la transaccion
10:47:43 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x000001B769D67590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
10:47:43 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x000001B7680E8850>
10:57:43 AM: DEBUG [cursor_del_pool.py: 11] Inicio de with metodo __enter__ 
10:57:43 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001E572964B80>
10:57:43 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x000001E5725B7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
10:57:43 AM: DEBUG [persona_dao.py: 19] b'SELECT * FROM persona ORDER BY id_persona'
10:57:43 AM: DEBUG [cursor_del_pool.py: 18] Se ejecuta el metodo __exit__()
10:57:43 AM: DEBUG [cursor_del_pool.py: 25] Commit de la transaccion
10:57:43 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x000001E5725B7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
10:57:43 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x000001E572964B80>
10:57:43 AM: DEBUG [persona_dao.py: 61] Id Persona: 1, Nombre: Juan, Apellido: Perez, Email: jperez@mail.com 
10:57:43 AM: DEBUG [persona_dao.py: 61] Id Persona: 2, Nombre: Karla1, Apellido: Gomez2, Email: Kgomez21@mail.com 
10:57:43 AM: DEBUG [persona_dao.py: 61] Id Persona: 4, Nombre: Carlos, Apellido: Reinoza, Email: Creinoza@mail.com 
10:57:43 AM: DEBUG [persona_dao.py: 61] Id Persona: 11, Nombre: Maria, Apellido: Longa, Email: Mlonga@mail.com 
10:57:43 AM: DEBUG [persona_dao.py: 61] Id Persona: 12, Nombre: Fernando, Apellido: Ibarra, Email: Fibarra@mail.com 
10:57:43 AM: DEBUG [persona_dao.py: 61] Id Persona: 13, Nombre: Marcos, Apellido: Tunez, Email: Mtunez@mail.com 
11:01:18 AM: DEBUG [cursor_del_pool.py: 11] Inicio de with metodo __enter__ 
11:01:19 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000014F57B08F10>
11:01:19 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x0000014F57AA7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:01:19 AM: DEBUG [persona_dao.py: 32] b'INSERT INTO persona(nombre, apellido, email) VALUES(%s,%S,%S)'
11:01:19 AM: DEBUG [persona_dao.py: 33] Persona a insertar: Id Persona: None, Nombre: Pedro, Apellido: Najera, Email: Pnajera@mail.com 
11:01:19 AM: DEBUG [cursor_del_pool.py: 18] Se ejecuta el metodo __exit__()
11:01:19 AM: DEBUG [cursor_del_pool.py: 22] Ocurrio una excepcion: unsupported format character 'S' (0x53) at index %Id
11:01:19 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x0000014F57AA7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:01:19 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x0000014F57B08F10>
11:07:49 AM: DEBUG [cursor_del_pool.py: 11] Inicio de with metodo __enter__ 
11:07:49 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000002A0D9588F10>
11:07:49 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x000002A0D9527590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:07:49 AM: DEBUG [persona_dao.py: 41] b'UPDATE persona SET nombre=%s, apellido=%s, email=%s WHERE id_persona=%s'
11:07:49 AM: DEBUG [persona_dao.py: 42] Persona a actualizar: Id Persona: 1, Nombre: Juan3, Apellido: Perez2, Email: jperez1@mail.com 
11:07:49 AM: DEBUG [cursor_del_pool.py: 18] Se ejecuta el metodo __exit__()
11:07:49 AM: DEBUG [cursor_del_pool.py: 25] Commit de la transaccion
11:07:49 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x000002A0D9527590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:07:49 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x000002A0D9588F10>
11:07:49 AM: DEBUG [persona_dao.py: 71] Personas actualizadas: 1
11:09:18 AM: DEBUG [cursor_del_pool.py: 11] Inicio de with metodo __enter__ 
11:09:18 AM: DEBUG [conexion.py: 28] Creacion de Pool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000002A63D808F40>
11:09:18 AM: DEBUG [conexion.py: 40] Conexion obtenida del Pool: <connection object at 0x000002A63D7A7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:09:18 AM: DEBUG [persona_dao.py: 50] b'DELETE FROM persona WHERE id_persona = %s'
11:09:18 AM: DEBUG [persona_dao.py: 51] Persona a eliminar: Id Persona: 13, Nombre: None, Apellido: None, Email: None 
11:09:19 AM: DEBUG [cursor_del_pool.py: 18] Se ejecuta el metodo __exit__()
11:09:19 AM: DEBUG [cursor_del_pool.py: 25] Commit de la transaccion
11:09:19 AM: DEBUG [conexion.py: 47] Regresamos la conexion al pool: <connection object at 0x000002A63D7A7590; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
11:09:19 AM: DEBUG [conexion.py: 48] Estado del pool: <psycopg2.pool.SimpleConnectionPool object at 0x000002A63D808F40>
11:09:19 AM: DEBUG [persona_dao.py: 76] Personas eliminadas: 1
